content/php/concepts/switch/switch.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Switch'
+Description: 'Switch case statements allow the execution of different blocks of code based on the value of a variable or expression.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Switch'
+  - 'Conditionals'
```

Comment:
This tag (Conditionals) will need to be added to the [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) file. You can include the edit in this PR.